### PR TITLE
Tidy

### PR DESCRIPTION
**Pull request by edunham**
only remaining error is that it wants a libc version pinned rather than just an asterisk. What should that version be?

---

This change is [Reviewable](https://reviewable.io/reviews/servo/rust-freetype/39)

### CODE REVIEW

**Comment by SimonSapin** [comment]
Please use `libc = "0.2"` in Cargo.toml.

File: configure [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-SRCDIR="$(cd $(dirname $0) && pwd)"
+SRCDIR = "$(cd $(dirname $0) && pwd)"
```
Comment (jdm) [inline configure]: This change isn't valid.

File: src/freetype.rs [hunk 2]
```diff
@@ -20,7 +20,10 @@ pub type FT_UFast = c_uint;
 pub type FT_Memory = *mut struct_FT_MemoryRec_;
 pub type FT_Alloc_Func = extern fn(mem: FT_Memory, size: c_long) -> *mut c_void;
 pub type FT_Free_Func = extern fn(mem: FT_Memory, block: *mut c_void);
-pub type FT_Realloc_Func = extern fn(mem: FT_Memory, cur_size: c_long, new_size: c_long, block: *mut c_void) -> *mut c_void;
+pub type FT_Realloc_Func = extern fn(mem: FT_Memory,
+                                          cur_size: c_long,
```
Comment (jdm) [inline src/freetype.rs]: Could these be indented to match the first argument?

**Comment by edunham** [comment]
fixed identified issues and rebased

**Comment by emilio** [comment]
@bors-servo r+

Thanks Emily :)

**Pull request merged**